crates/sui-framework/packages/deepbook/sources/custodian_v2.move

```diff
@@ -27,19 +27,20 @@ module deepbook::custodian_v2 {
     // Custodian for limit orders.
     struct Custodian<phantom T> has key, store {
         id: UID,
-        /// Map from the owner address of  AccountCap object to an Account object
+        /// Map from the owner address of AccountCap object to an Account object
         account_balances: Table<address, Account<T>>,
     }
 
     /// Create an `AccountCap` that can be used across all DeepBook pool
-    public(friend) fun mint_account_cap(owner: address, ctx: &mut TxContext): AccountCap {
+    public(friend) fun mint_account_cap(ctx: &mut TxContext): AccountCap {
         AccountCap {
             id: object::new(ctx),
-            owner
+            owner: sender(ctx)
         }
     }
 
-    public fun account_owner(accountCap: &AccountCap): address {
+    /// Return the owner address of an AccountCap
+    public fun account_cap(accountCap: &AccountCap): address {
```

Comment:
- Can we keep the old function name `account_owner`? The name makes sense--it takes an account cap and gives you an owner address.
-  Nit: `accountCap` -> `account_cap`

Reply:
Oh, I misunderstood.